Widget/Fitness/FitnessWidget.swift

```diff
@@ -0,0 +1,238 @@
+//
+//  FitnessWidget.swift
+//  PennMobile
+//
+//  Created by Pulkith Paruchuri on 10/8/23.
+//  Copyright © 2023 PennLabs. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import PennMobileShared
+import WidgetKit
+
+extension ConfigureFitnessWidgetIntent: ConfigurationRepresenting {
+    struct Configuration {
+        let background: WidgetBackgroundType
+        let complex: FitnessChosenComplex
+    }
+    
+    var configuration: Configuration {
+        return Configuration(background: background, complex: complex)
+    }
+}
+
+/* Code from FitnessRoomRow */
+private func lastUpdatedFormattedTime (date: Date) -> String {
+    let interval = -date.timeIntervalSinceNow
+    let hours = Int(interval / 3600)
+    let minutes = Int((interval - 3600 * Double(hours)) / 60)
+    return "Updated \(hours)h \(minutes)m ago"
```

Comment:
I changed this slightly since this PR was made, can you make it consistent with the other fitness code?